.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/tonymillion/Reachability.git
 [submodule "spec/cocoapods-integration-specs"]
 	path = spec/cocoapods-integration-specs
-	url = https://github.com/CocoaPods/cocoapods-integration-specs.git
+	url = https://github.com/benasher44/cocoapods-integration-specs.git
```

Comment:
well you have push now so you can just make your branch in the CocoaPods repo, I thought you'd done that